packages/app-root/src/app/style-registry.js

```diff
@@ -0,0 +1,27 @@
+'use client'
```

Comment:
Interesting, so the server sends an unstyled page to the browser, then styles are built in the client?

Reply:
Or maybe not. The downloaded HTML includes an inline stylesheet. Vendor prefixes can definitely be removed here.

If we could move all this into an external CSS file, that would cost us less to serve, but I don't think that's possible with styled components and Next.js.
<img width="600" alt="View Source for the new app, showing an inline <style> stylesheet in the page head. " src="https://github.com/zooniverse/front-end-monorepo/assets/59547/c255fa69-f7cb-40aa-bdcc-f12b2b637d42">

Reply:
👍 this stylesheet fixes the page styling issue.

Reply:
Breakpoints should be `48rem`, not `768px`, but that's probably a bug in the Grommet theme.

Reply:
This is odd. I built the app a second time. After the second build, I don't see the server-side stylesheet in the HTML, so it's unstyled again when it first loads.


https://github.com/zooniverse/front-end-monorepo/assets/59547/287e207a-11c0-47d3-8f0e-196877c52102

Reply:
Seems ok after running the bootstrap script and building for a third time. 🤷‍♂️

Reply:
Good to know, thanks for double checking!